src/admin/controller/api/system.js

```diff
@@ -91,24 +90,25 @@ export default class extends base {
         });
 
       /** 安装依赖 */
-      case '3':
+      case '3': {
```

Comment:
http://eslint.org/docs/rules/no-case-declarations

This rule disallows lexical declarations (let, const, function and class) in case/default clauses. The reason is that the lexical declaration is visible in the entire switch block but it only gets initialized when it is assigned, which will only happen if the case where it is defined is reached.

To ensure that the lexical declaration only applies to the current case clause wrap your clauses in blocks.